bird-slideshow.py

```diff
@@ -89,7 +89,7 @@ def _read_config(self):
                 elif name == "default_resolution":
                     self.win_start_res = value
                 elif name == "max_grow":
```

Comment:
Shouldn't this also change to "max_size" here?

Reply:
that is, shouldn't the conditional be: elif name == "max_size"

Reply:
Yes it should be.